scripts/patchup-example-names.mjs

```diff
@@ -1,17 +1,29 @@
 import fs from "fs/promises";
 import path from "path";
+import sort from "sort-package-json";
 
-let dirPath = path.join(process.cwd(), "examples");
-let dir = await fs.readdir(dirPath);
+let examplePath = path.join(process.cwd(), "examples");
+let templatePath = path.join(process.cwd(), "templates");
+let examples = await fs.readdir(examplePath);
+let templates = await fs.readdir(templatePath);
 
-dir.forEach(async (name) => {
-  let fullPath = path.join(dirPath, name);
+async function update(dir, name, base) {
+  let fullPath = path.join(dir, name);
   let stat = await fs.stat(fullPath);
   if (stat.isDirectory()) {
     let pkgPath = path.join(fullPath, "package.json");
     let pkg = await fs.readFile(pkgPath);
     let obj = JSON.parse(pkg);
-    obj.name = `remix-example-${name}`;
-    await fs.writeFile(pkgPath, JSON.stringify(obj, null, 2) + "\n");
+    obj.name = `remix-${base}-${name}`;
+    let sorted = sort(obj);
```

Comment:
@mcansh Not sure if it's a good idea to add running `sort-package-json` in this PR tbh as I think it could create some conflicts with #2200 (which is merged in `main`) 🤔

Reply:
yeah i think we can make a follow up PR after we get everything tightened up and squared away for re-adding that to this